src/SFA.DAS.Tools.Support.UnitTests/AutoFixture/AutoMoqDataAttribute.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using AutoFixture;
 using AutoFixture.AutoMoq;
-using AutoFixture.Xunit2;
+using AutoFixture.NUnit3;
 
 namespace SFA.DAS.Tools.Support.UnitTests.AutoFixture
 {
-    public class AutoMoqDataAttribute : AutoDataAttribute
+    public class MoqAutoDataAttribute : AutoDataAttribute
     {
-        public AutoMoqDataAttribute()
+        public MoqAutoDataAttribute()
```

Comment:
this shouldn't be required. Can use `SFA.DAS.Testing.AutoFixture` for `MoqAutoData` attribute.